dogs.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="mystyle.css">
+</head>
+<body>
+
+  <div class="header">
+    <h1>My Website</h1>
+    <img src="./images/adoptadog-logo.svg" alt="Logo" width="1500" height="800">
+    <a href="./forms.html" class="button">Checkout</a>
+    <p>A website created by me.</p>
+  </div>
+
+<div class="navbar">
```

Comment:
instead of a div with a class of navbar, think of an HTML5 element that is good for nav type things. ;)